includes/polling/os/unix.inc.php

```diff
@@ -120,6 +120,25 @@
     if (!$features) {
         $features = 'GENERIC';
     }
+
+    # Try detect using the extended option (dmidecode)
+    $version_dmide = str_replace("\"", "", snmp_get($device, ".1.3.6.1.4.1.2021.7890.2.4.1.2.8.104.97.114.100.119.97.114.101.1", "-Oqv"));
```

Comment:
Are all these str_replace() calls necessary?  The code above is legacy code, snmp_get() should have clean output.

Reply:
Ah good catch, I took the existing code for Linux distributions and adapted it for FreeBSD. Have just done a bit of testing and the code does work just fine without str_replace(). I will push a new commit that addresses my patch momentarily.

The rest of the file in question is similarly afflicted so I'll go through and clean that up and push a second commit for that also.